README.md

```diff
@@ -13,29 +13,29 @@
   <img width="300" height="150" src="./assets/delete.gif" alt="Areas delete">
 </p>
 
-Areas aims to provide a flexible and customizable environment to create custom UI layouts using [Vue.js](https://vuejs.org/).
+Areas aim to provide a flexible and customizable environment to create custom UI layouts using [Vue.js](https://vuejs.org/).
```

Comment:
Areas is singular, this is the name of the component, not some areas.

Reply:
Sorry for my mistake.